monai/fl/client/monai_algo.py

```diff
@@ -80,55 +68,38 @@ def compute_weight_diff(global_weights, local_var_dict):
     return weight_diff
 
 
-def check_bundle_config(parser):
-    for k in RequiredBundleKeys:
-        if parser.get(k, None) is None:
-            raise KeyError(f"Bundle config misses required key `{k}`")
-
-
-def disable_ckpt_loaders(parser):
-    if BundleKeys.VALIDATE_HANDLERS in parser:
-        for h in parser[BundleKeys.VALIDATE_HANDLERS]:
-            if ConfigComponent.is_instantiable(h):
-                if "CheckpointLoader" in h["_target_"]:
-                    h["_disabled_"] = True
-
-
 class MonaiAlgoStats(ClientAlgoStats):
     """
     Implementation of ``ClientAlgoStats`` to allow federated learning with MONAI bundle configurations.
 
     Args:
-        bundle_root: path of bundle.
```

Comment:
@Nic-Ma, I feel we should keep the bundle_root argument and then build the train/eval workflows as you do in the testing scripts (`ConfigWorkflow(os.path.join(_data_dir, "config_fl_train.json")`). One reason is that FL platforms like nvflare are often config-based, and passing Python objects directly is not straightforward.

Reply:
This will also minimize the changes needed on the nvflare integration side. [Here](https://github.com/NVIDIA/NVFlare/blob/b0de532fc2f65de30eb8af728561bab33fdaf076/integration/monai/examples/spleen_ct_segmentation_sim/job/app/config/config_fed_client.json#L28) is an example of how MonaiAlgo is configured in nvflare.

Reply:
Sure, added these parameters back in the latest commit.

Thanks.